### PR TITLE
adds objc_show method for Objective-C compatibility

### PR DESCRIPTION
**Pull request by markedwardmurray**
Resolves issue #9 

Hey Kevin! Thanks so much for contributing this library to open source. It's very well written and easy to use. In the process of including it in a project written primarily in Objective-C, I ended up writing the `objc_show` method that you had described in issue #9 and it seems to be working just fine so far (though I haven't been making use of the values returned by the `show()` method).

Cheers!
~Mark

### CODE REVIEW

File: DropDown/src/DropDown.swift [hunk 1]
```diff
@@ -488,6 +488,23 @@ extension DropDown {
...
+    @objc(show)
+    public func objc_show() -> NSDictionary {
+        let (canBeDisplayed, offScreenHeight) = show()
+        var info = [NSObject : AnyObject]()
+        info["canBeDisplayed"] = canBeDisplayed
+        
+        if let offScreenHeight = offScreenHeight {
+            info["offScreenHeight"] = offScreenHeight
+        }
+        return NSDictionary(dictionary: info)
```
Comment (kevin-hirsch) [inline DropDown/src/DropDown.swift]: Could you put a new line before the `return` to follow the code convention of this project, please ? :) 👍

**Comment by kevin-hirsch** [comment]
Hello @markedwardmurray !
Thanks for this PR :) 👍

File: DropDown/src/DropDown.swift (hunk 1 above)
Comment (markedwardmurray) [inline DropDown/src/DropDown.swift]: Oops! Wrote that in the wee morning hours. Just amended the commit for spacing 😁

**Comment by markedwardmurray** [comment]
thanks for accepting @kevin-hirsch :grin:

**Comment by kevin-hirsch** [comment]
Thanks to you ! 👍

**Pull request merged**